src/helpers/utils/filter-functions.js

```diff
@@ -0,0 +1,45 @@
+const sortByPrice = (filters, data) => {
+    switch (filters.sortByPrice) {
+        case 'lth':
+            return [...data].sort((a, b) => a.price - b.price)
```

Comment:
writing "firstItem" and "secondItem" instead of 'a' and 'b' would give more meaning to the code.

Reply:
that is sort won't make a diff anyway